### ISSUE TITLE
error on validation lower than on training for aspirin dataset

### ISSUE THREAD

**Issue by heliogabale** [mir-group/allegro#56]
Hello
I am using the config /example.yml to train a model on the aspirin dataset.
The error (MAE and RMSE) I am getting on the validation set on energy is 5x-10x lower than the one on the training set (MAE).
The dataset is shuffled and it uses a random split train/validation .
This is true  throughout the training ( I am stopping for now at 100 epochs).

**Comment by cw-tan** [comment]
This is pretty old, but it's likely because of the use of EMA (i.e. exponential moving average of weights for validation).
